---
title: Love Lab @ UNC-Chapel Hill
layout: page
---

<img src="assets/well.png" ALT="picture of the well on UNC campus, from unc.edu"/>

<!-- {% include JB/setup %} -->

Genetics Department ~
Biostatistics Department ~
UNC-Chapel Hill ~ 
see [lab members](pages/lab.html) for contact details.

The Love Lab on GitHub:

* <https://github.com/thelovelab/>

### News

* October 2024 - Justin's first R package *plyxp* is released on Bioconductor <https://github.com/jtlandis/plyxp>
* August 2024 - Yuriko's first preprint from the lab: *ClassifyGxT* [doi: 10.1101/2024.08.03.605142](https://doi.org/10.1101/2024.08.03.605142), with associated software <https://github.com/yharigaya/classifygxt>
* July 2024 - Ji-Eun successfully defends her PhD thesis. Congrats Dr Park!
* May 2024 - Justin Landis joins the Love lab as a BCB PhD candidate
* April 2024 - Isabella successfully defends her senior honors thesis. Congrats Isa!
* March 2024 - Ji-Eun's statistical model *DiffSig* is published: [doi: 10.1158/1055-9965.EPI-23-0728](https://doi.org/10.1158/1055-9965.EPI-23-0728)
* January 2024 - Wancen posts *GPlag* preprint: [doi: 10.48550/arXiv.2401.07400](https://doi.org/10.48550/arXiv.2401.07400)
* January 2024 - Wancen successfully defends her PhD thesis. Congrats Dr. Mu!
* September 2023 - *Tidyomics* preprint is posted: [doi: 10.1101/2023.09.10.557072](https://doi.org/10.1101/2023.09.10.557072)

### Research

The Love Lab uses statistical models to infer biologically meaningful
patterns in high-dimensional datasets, and develops
open-source statistical software for the [Bioconductor Project](https://bioconductor.org).
At UNC-Chapel Hill, we often collaborate with groups in the 
*Genetics Department* and the *Lineberger Comprehensive Cancer Center*, 
studying how genetic variants relevant to diseases are associated with changes
in molecular and cellular phenotypes.

* We collaborate with the 
  [Won lab](https://www.wonlab.org/) and the 
  [Mohlke lab](https://mohlke.web.unc.edu/) as part of
  the IGVF consortium, studying the effect of disease-associated GWAS
  variants via *in vivo* massively parallel reporter assays (MPRA) to
  assess tissue-, sex-, and treatment-specific allelic effects.
* We collaborate with the [Patro lab](http://www.robpatro.com/) in the
  development of computational and statistical methods for
  quantification of RNA transcript abundance, including the software
  packages [Salmon](https://combine-lab.github.io/salmon/)
  and [tximeta](https://bioconductor.org/packages/tximeta).
* We collaborate with the [Stein lab](https://steinlab.org) in the
  study of genetic risk for neuropsychiatric disorders, leveraging
  data from molecular assays of chromatin accessibility and the
  transcriptome in neural progenitor cell lines to macroscale
  phenotypes such as gross human brain structure measured with MRI.
  In collaboration with Stein lab researchers, we have developed the 
  [MRLocus](https://mikelove.github.io/mrlocus) method for identifying
  causal genes from QTL studies and GWAS.
* We collaborate with the
  [Troester lab](https://sph.unc.edu/adv_profile/melissa-troester-phd/) 
  and the Carolina Breast Cancer Study, investigating the intersection
  between gene expression profiles of breast tumors and genetic risk
  for breast cancer, with a focus on racial disparities in patient
  outcomes.
* We develop, support, and maintain the
  [DESeq2](http://bioconductor.org/packages/DESeq2) package
  for statistical analysis of RNA-seq and other sequencing experiments,
  and associated packages and workflows for analyzing genomics
  data. See the [Software](pages/software.html) tab for more details.

### Past news items

2023

* Mike starts as Associate Director of the Bioinformatics and Computational Biology 
  (BCB) PhD program at UNC-Chapel Hill
* Euphy's paper on detecting allelic imbalance at the isoform-level is published: <br/>
  Euphy Y. Wu, Noor P. Singh, Kwangbom Choi, Mohsen Zakeri, Matthew Vincent, 
  Gary A. Churchill, Cheryl L. Ackert-Bicknell, Rob Patro & Michael I. Love <br/>
  "SEESAW: detecting isoform-level allelic imbalance accounting for inferential uncertainty" <br/>
  [doi: 10.1186/s13059-023-03003-x](https://doi.org/10.1186/s13059-023-03003-x)
* Wancen's *bootRanges* and Eric's *matchRanges* methods published: <br/>
  Mu *et al.* "bootRanges: flexible generation of null sets of genomic ranges for hypothesis testing" <br/>
  [doi: 10.1093/bioinformatics/btad190](https://doi.org/10.1093/bioinformatics/btad190) <br/>
  Davis *et al.* "matchRanges: generating null hypothesis genomic ranges via covariate-matched sampling" <br/>
  [doi: 10.1093/bioinformatics/btad197](https://doi.org/10.1093/bioinformatics/btad197)
* Ji-Eun's preprint, *Diffsig*, a
  method for associating risk factors with mutational signatures: <br/>
  [doi: 10.1101/2023.02.09.527740](https://doi.org/10.1101/2023.02.09.527740)


2022

* Wancen Mu posts
  *bootRanges* preprint to *bioRxiv*: <br/>
  [doi: 10.1101/2022.09.02.506382](https://doi.org/10.1101/2022.09.02.506382)
* Euphy Wu posts her first preprint, on isoform-level
  allelic imbalance, to *bioRxiv*: <br/>
  [doi: 10.1101/2022.08.12.503785](https://doi.org/10.1101/2022.08.12.503785)
* Eric Davis (member of Phanstiel lab) posts
  *matchRanges* preprint to *bioRxiv*: <br/>
  [doi: 10.1101/2022.08.05.502985](https://doi.org/10.1101/2022.08.05.502985)
* Sarah Reifeis's preprint, on assessing etiologic
  heterogeneity in two phase studies with outcome dependent samples,
  posted to *medRxiv*: <br/>
  [doi: 10.1101/2022.07.20.22277805](https://doi.org/10.1101/2022.07.20.22277805)
* Wancen's first paper, on allelic imbalance in
  scRNA-seq published in *Bioinformatics*: <br/>
  [doi: 10.1093/bioinformatics/btac212](https://doi.org/10.1093/bioinformatics/btac212)

2021

* [nullranges](https://nullranges.github.io/nullranges)
  is released on Bioconductor! Joint work from Wancen and Eric Davis
  from Doug Phanstiel lab.
* Wancen's [airpart](https://bioconductor.org/packages/airpart)
  method now has a preprint on *bioRxiv*:
  [doi: 10.1101/2021.10.15.464546](https://doi.org/10.1101/2021.10.15.46454)
* Congratulations to Sean McCabe for winning the 2021 Larry
  Kupper Award for Best Dissertation Publication! Congratulations to
  Sarah Reifeis for being inducted into Delta Omega, and on receiving
  a Distinguished Student Paper Awards for ENAR 2021!
* Wancen's single cell allelic analysis package accepted
  into Bioconductor:
  [airpart](https://bioconductor.org/packages/airpart)
* Anqi and Nana's MRLocus paper is published in *PLOS Genetics*:
  [doi: 10.1371/journal.pgen.1009455](https://doi.org/10.1371/journal.pgen.1009455)
* Arjun's MOSTWAS paper is published in *PLOS Genetics*:
  [doi: 10.1371/journal.pgen.1009398](https://doi.org/10.1371/journal.pgen.1009398)
* Arjun's DeCompress paper is published in *Nucleic Acids Research*:
  [doi: 10.1093/nar/gkab031](https://doi.org/10.1093/nar/gkab031)
* Scott's inferential summaries for scRNA-seq paper is published in
  *Bioinformatics*:
  [doi: 10.1093/bioinformatics/btab001](https://doi.org/10.1093/bioinformatics/btab001)


2020

* Kwame won a poster award at ABRCMS 2020, and a JTech
  CSHL Biological Data Science 2020 Scholarship. His poster and
  abstract were for his R package: [vizWithSCE](https://kwameforbes.github.io/vizWithSCE/).
* Anqi's last chapter is preprinted: "MRLocus: identifying causal genes
  mediating a trait through Bayesian estimation of allelic heterogeneity"
  [doi: 10.1101/2020.08.14.250720](https://doi.org/10.1101/2020.08.14.250720)
* Arjun's co-authored paper on NanoString normalization with Alina
  Hamilton is published in *Briefings in Bioinformatics*,
  [doi: 10.1093/bib/bbaa163](https://doi.org/10.1093/bib/bbaa163)
* Sarah Reifeis's first paper is published in *Genetic Epidemiology*:
  "Assessing exposure effects on gene expression"
  [doi: 10.1002/gepi.22324](https://doi.org/10.1002/gepi.22324)
* Arjun Bhattacharya's second and third papers are preprinted:
    - "An approach for normalization and quality control for NanoString RNA expression data"
  [doi: 10.1101/2020.04.08.032490](https://doi.org/10.1101/2020.04.08.032490)
    - "Multi-omic strategies for transcriptome-wide prediction and association studies"
  [doi: 10.1101/2020.04.17.047225](https://doi.org/10.1101/2020.04.17.047225)
* Tximeta is published in *PLOS Computational Biology*
  [doi: 10.1371/journal.pcbi.1007664](https://doi.org/10.1371/journal.pcbi.1007664)
* Arjun Bhattacharya's breast cancer TWAS is published in *Genome Biology*
  [doi: 10.1186/s13059-020-1942-6](https://doi.org/10.1186/s13059-020-1942-6)
* New paper on *F1000Research*: From first author Stuart Lee (Monash U.),
  "Fluent genomics with plyranges and tximeta"
  [doi: 10.12688/f1000research.22259.1](https://doi.org/10.12688/f1000research.22259.1)

2019

* Sean McCabe's second paper is preprinted:
  "ACTOR: a latent Dirichlet model to compare expressed 
  isoform proportions to a reference panel"
  [doi: 10.1101/856401](https://doi.org/10.1101/856401)
* Joshua Zitovsky's first paper is up on F1000Research:
  "Fast effect size shrinkage software for beta-binomial 
  models of allelic imbalance"
  [doi: 10.12688/f1000research.20916.1](https://doi.org/10.12688/f1000research.20916.1)
* Anqi Zhu's second paper published in *Nucleic Acids Research*:
  [doi: 10.1093/nar/gkz622](https://doi.org/10.1093/nar/gkz622),
  which describes the Swish method in the [fishpond](https://bioconductor.org/packages/fishpond) package.
* Sean McCabe's first paper published in *Briefings in Bioinformatics*:
  [doi: 10.1093/bib/bbz070](https://doi.org/10.1093/bib/bbz070),
  [Author Manuscript](../assets/mccabe_2019.pdf)

<a rel="me" href="https://genomic.social/@mikelove">@mikelove@genomic.social</a>
