---
title: Love Lab
layout: page
---

<!-- {% include JB/setup %} -->

<img style="float: right;" src="assets/michaellove.jpg">

<p style="margin-bottom: 0px"> PI: Michael I. Love </p>
<p style="color:grey; margin-bottom: 5px"> (he/him/his) </p>
Assistant Professor <br>
[Department of Biostatistics](http://sph.unc.edu/bios/biostatistics/) <br>
[Department of Genetics](http://www.med.unc.edu/genetics/) <br>
Member of [Lineberger Comprehensive Cancer Center](https://unclineberger.org/) <br>
University of North Carolina-Chapel Hill

[@mikelove](http://twitter.com/mikelove) ---
[Google Scholar](https://scholar.google.com/citations?user=vzXv764AAAAJ) ---
[Biostatistics Dept profile](http://sph.unc.edu/adv_profile/michael-love-phd/)

michaelisaiahlove at gmail dot com *

\* for software questions, do not email, instead use the [Bioconductor support site](https://support.bioconductor.org)

### News

* May 2021 - Congratulations to Sean McCabe for winning the 2021 Larry
  Kupper Award for Best Dissertation Publication! Congratulations to
  Sarah Reifeis for being inducted into Delta Omega, and on receiving
  a Distinguished Student Paper Awards for ENAR 2021!
* April 2021 - Wancen's single cell allelic analysis package accepted
  into Bioconductor:
  [airpart](https://bioconductor.org/packages/airpart)
* April 2021 - Anqi and Nana's MRLocus paper is published in *PLOS Genetics*:
  [doi: 10.1371/journal.pgen.1009455](https://doi.org/10.1371/journal.pgen.1009455)
* March 2021 - Arjun's MOSTWAS paper is published in *PLOS Genetics*:
  [doi: 10.1371/journal.pgen.1009398](https://doi.org/10.1371/journal.pgen.1009398)
* January 2021 - Arjun's DeCompress paper is published in *Nucleic Acids Research*:
  [doi: 10.1093/nar/gkab031](https://doi.org/10.1093/nar/gkab031)
* January 2021 - Scott's inferential summaries for scRNA-seq paper is published in
  *Bioinformatics*:
  [doi: 10.1093/bioinformatics/btab001](https://doi.org/10.1093/bioinformatics/btab001)
* November 2020 - Kwame won a poster award at ABRCMS 2020, and a JTech
  CSHL Biological Data Science 2020 Scholarship. His poster and
  abstract were for his R package: [vizWithSCE](https://kwameforbes.github.io/vizWithSCE/).

### Research

The Love Lab uses statistical models to infer biologically meaningful
patterns in high-dimensional datasets, and develops
open-source statistical software for the [Bioconductor Project](https://bioconductor.org).
At UNC-Chapel Hill, we often collaborate with groups in the 
*Genetics Department* and the *Lineberger Comprehensive Cancer Center*, 
studying how genetic variants relevant to diseases are associated with changes
in molecular and cellular phenotypes.

* We collaborate with the [Patro lab](http://www.robpatro.com/) in the
  development of computational and statistical methods for
  quantification of RNA transcript abundance, including the software
  packages [Salmon](https://combine-lab.github.io/salmon/)
  and [tximeta](https://bioconductor.org/packages/tximeta).
* We collaborate with the [Stein lab](https://steinlab.org) in the
  study of genetic risk for neuropsychiatric disorders, leveraging
  data from molecular assays of chromatin accessibility and the
  transcriptome in neural progenitor cell lines to macroscale
  phenotypes such as gross human brain structure measured with MRI.
* We collaborate with the
  [Troester lab](https://sph.unc.edu/adv_profile/melissa-troester-phd/) 
  and the Carolina Breast Cancer Study, investigating the intersection
  between gene expression profiles of breast tumors and genetic risk
  for breast cancer, with a focus on racial disparities in patient
  outcomes.
* We develop, support, and maintain the
  [DESeq2](http://bioconductor.org/packages/DESeq2) package
  for statistical analysis of RNA-seq and other sequencing experiments,
  and associated packages and workflows for analyzing genomics
  data. See the [Software](pages/software.html) tab for more details.

### PI - Background

From 2013-2016, I was a Postdoctoral Fellow in the group of
Rafael Irizarry in the Biostatistics and Computational Biology
Department at the Dana-Farber Cancer Institute and the Harvard
TH Chan School of Public Health.
I completed a PhD in Computational Biology and Scientific Computing
(2013) in the Vingron Department at the Max Planck Institute for
Molecular Genetics in Berlin and the Mathematics and Informatics
Department of the Freie Universit√§t, Berlin.  I completed a Statistics
M.S. (2010) and Mathematics B.S. (2005) at Stanford University.

### Past news items

2020

* Anqi's last chapter is preprinted: "MRLocus: identifying causal genes
  mediating a trait through Bayesian estimation of allelic heterogeneity"
  [doi: 10.1101/2020.08.14.250720](https://doi.org/10.1101/2020.08.14.250720)
* Arjun's co-authored paper on NanoString normalization with Alina
  Hamilton is published in *Briefings in Bioinformatics*,
  [doi: 10.1093/bib/bbaa163](https://doi.org/10.1093/bib/bbaa163)
* Sarah Reifeis's first paper is published in *Genetic Epidemiology*:
  "Assessing exposure effects on gene expression"
  [doi: 10.1002/gepi.22324](https://doi.org/10.1002/gepi.22324)
* Arjun Bhattacharya's second and third papers are preprinted:
    - "An approach for normalization and quality control for NanoString RNA expression data"
  [doi: 10.1101/2020.04.08.032490](https://doi.org/10.1101/2020.04.08.032490)
    - "Multi-omic strategies for transcriptome-wide prediction and association studies"
  [doi: 10.1101/2020.04.17.047225](https://doi.org/10.1101/2020.04.17.047225)
* Tximeta is published in *PLOS Computational Biology*
  [doi: 10.1371/journal.pcbi.1007664](https://doi.org/10.1371/journal.pcbi.1007664)
* Arjun Bhattacharya's breast cancer TWAS is published in *Genome Biology*
  [doi: 10.1186/s13059-020-1942-6](https://doi.org/10.1186/s13059-020-1942-6)
* New paper on *F1000Research*: From first author Stuart Lee (Monash U.),
  "Fluent genomics with plyranges and tximeta"
  [doi: 10.12688/f1000research.22259.1](https://doi.org/10.12688/f1000research.22259.1)

2019

* Sean McCabe's second paper is preprinted:
  "ACTOR: a latent Dirichlet model to compare expressed 
  isoform proportions to a reference panel"
  [doi: 10.1101/856401](https://doi.org/10.1101/856401)
* Joshua Zitovsky's first paper is up on F1000Research:
  "Fast effect size shrinkage software for beta-binomial 
  models of allelic imbalance"
  [doi: 10.12688/f1000research.20916.1](https://doi.org/10.12688/f1000research.20916.1)
* Anqi Zhu's second paper published in *Nucleic Acids Research*:
  [doi: 10.1093/nar/gkz622](https://doi.org/10.1093/nar/gkz622),
  which describes the Swish method in the [fishpond](https://bioconductor.org/packages/fishpond) package.
* Sean McCabe's first paper published in *Briefings in Bioinformatics*:
  [doi: 10.1093/bib/bbz070](https://doi.org/10.1093/bib/bbz070),
  [Author Manuscript](../assets/mccabe_2019.pdf)
