---
layout: page
title: "Lab members and RAs"
---

<style>
table{
  border-collapse: collapse;
  border-spacing: 0;
}

th{
  background-color: #eeeeee;
  padding: 5px;
}

td{
  border: 1px solid #eeeeee;
  padding: 5px;
}
</style>

| Name | Research topics | |
| :--- | :---- | :---- |
| [Arjun Bhattacharya](https://bhattacharya-a-bt.github.io) [@bhattac_a_bt](https://twitter.com/bhattac_a_bt) | TWAS in Carolina Breast Cancer Study; <br> Deconvolution of tumor expression assayed via Nanostring; <br> Co-supervised with Melissa Troester | <img width="100" src="../assets/arjunbhattacharya.jpg"> | 
| Sarah Reifeis [@sarah_reifeis](https://twitter.com/sarah_reifeis) | Exposure effects on gene expression in observational studies; <br> Co-supervised with Michael Hudgens | <img width="100" src="../assets/sarahreifeis.jpg"> |
| Scott Van Buren | Uncertainty estimation in scRNA-seq; <br> Co-supervised with Naim Rashid | <img width="100" src="../assets/scottvanburen.jpg"> |
| Euphy Wu | Allele-specific expression estimation in RNA-seq | <img width="100" src="../assets/euphywu.jpg"> |
| Wancen Mu | Allele-specific expression in scRNA-seq | <img width="100" src="../assets/wancenmu.jpg"> |
| Alexandre Lockhart | Identification of mutational signatures via targeted sequencing | <img width="100" src="../assets/alexlockhart.jpg"> |
| Ji-Eun Park | Mutational signatures and risk factors in breast cancer; <br> Co-supervised with Di Wu | <img width="100" src="../assets/ji-eunpark.jpg"> |
| Kwame Forbes | Integration of single cell expression data for exploration of bulk DE analysis | <img width="100" src="../assets/kwameforbes.jpg"> |

## Alumni

| Name | Research topic | Current position &nbsp; &nbsp; &nbsp; | |
| :--- | :---- | :---- | :---- |
| [Sean McCabe](https://mccabes292.github.io/) [@SeanMcCabeBIOS](https://twitter.com/SeanMcCabeBIOS) | Multi-omics visualization: [movie](https://github.com/mccabes292/movie); <br> LDA of isoforms: [actor](https://github.com/mccabes292/actor) | Postdoc in Lin lab | <img width="100" src="../assets/seanmccabe.png"> |
| Anqi Zhu [@anqiz91](https://twitter.com/anqiz91) | Differential analysis of RNA-seq count data; <br> Developed [apeglm](http://bioconductor.org/packages/apeglm), [swish](http://bioconductor.org/packages/fishpond) | 23andme | <img width="100" src="../assets/anqizhu.png"> |
| Amanda Tapia | Colocalization of QTL and GWAS | Li lab member | <img width="100" src="../assets/amandatapia.jpg"> |
| Joshua Zitovsky | Fast implementation of *apeglm* for allelic expression ratios | Kosorok lab member | <img width="100" src="../assets/joshuazitovsky.jpg"> | 

## Friends of the lab

| Name | Research topic | School (lab) | |
| :--- | :---- | :---- | :---- |
| [Avi Srivastava](https://k3yavi.github.io/) [@k3yavi](https://twitter.com/k3yavi) | Fast mapping, scRNA-seq quantification | Stony Brook University (Patro) | <img width="100" src="../assets/avisrivastava.jpg"> |
| [Hirak Sarkar](https://hiraksarkar.github.io/) [@hiraksarkar](https://twitter.com/hiraksarkar) | Selective alignment, scRNA-seq simulation, transcript grouping | University of Maryland (Patro) | <img width="100" src="../assets/hiraksarkar.jpg"> |
