---
layout: page
title: "Lab members"
---

<style>
table{
  border-collapse: collapse;
  border-spacing: 0;
}

th{
  background-color: #eeeeee;
  padding: 5px;
}

td{
  border: 1px solid #eeeeee;
  padding: 5px;
}
</style>


**Michael I. Love** (he/him/his) <br/> 
Associate Professor - 
[Genetics](http://www.med.unc.edu/genetics/) and
[Biostatistics](http://sph.unc.edu/bios/biostatistics/)
at UNC-Chapel Hill <br/>
Associate Director - [Bioinformatics and Computational Biology (BCB) PhD program](https://bcb.unc.edu/)

[Google Scholar](https://scholar.google.com/citations?user=vzXv764AAAAJ) ~
[Curriculum Vitae](../assets/michael_love_cv.pdf)<br>
<a rel="me" href="https://genomic.social/@mikelove">@mikelove@genomic.social</a> ~
@mikelove.bsky.social

Email: michaelisaiahlove [at] gmail / love [at] unc.edu

For software support, please post to [Bioconductor support site](https://support.bioconductor.org).

---

| Name | Research topics | Position | <font color="#EEEEEE">~~~~~~~~~~</font> |
| :--- | :---- | :---- |
| Michael Love | Statistical software for genomic data analysis; <br> Causal mechanisms for GWAS loci | PI | <img width="100" src="../assets/michaellove.jpg"> |
| Jon Rosen | Functional interrogation of GWAS variants; <br> MPRA library design and analysis in IGVF and AMP-CMD; <br> [Gen. Epi. of HLB Traits](https://hlbgenepit32.web.unc.edu/), [Karen Mohlke lab](https://mohlke.web.unc.edu/)  | Post-doctoral | <img width="100" src="../assets/jonrosen.jpg"> |
| [Euphy Wu](https://www.linkedin.com/in/euphy-wu-09b14383) <br> [@FennecPaaaw](https://twitter.com/FennecPaaaw) | Isoform-level allelic imbalance: [SEESAW](https://mikelove.github.io/fishpond/articles/allelic.html); <br> Co-supervised with Naim Rashid | PhD candidate (Bios) | <img width="100" src="../assets/euphywu.jpg"> |
| [Ji-Eun Park](https://jennprk.github.io/) <br> [@JiEunPark](https://twitter.com/_JiEun_Park_) | Mutational signatures and risk factors in cancer; <br> Co-supervised with Di Wu | PhD candidate (Bios) | <img width="100" src="../assets/ji-eunpark.jpg"> |
| [Rachel Sharp](https://www.linkedin.com/in/rachel-sharp-a07b69272) <br> [@rrsharp122](https://twitter.com/rrsharp122) | Neuropsychiatric GWAS variants, GxE | PhD candidate (Neuro/BCB) | <img width="100" src="../assets/rachelsharp.jpg"> |
| Yuriko Harigaya | Statistical methods for response QTL <br> Co-supervised with William Valdar | PhD candidate (BCB) | <img width="100" src="../assets/yurikoharigaya.jpg"> |
| [Justin Landis](https://www.linkedin.com/in/justin-landis-64a586139) | Long-read RNA-seq methods | PhD candidate (BCB) | <img width="100" src="../assets/justinlandis.jpg"> |

## Alumni

### Doctoral students

| Name | Research topic | Current position | <font color="#EEEEEE">~~~~~~~~~~</font> |
| :--- | :---- | :---- | :---- |
| [Wancen Mu](https://wancen.github.io/) <br> [@wancenm](https://twitter.com/wancenm?lang=en) | Allelic imbalance in scRNA-seq: [airpart](https://bioconductor.org/packages/airpart); <br> Bootstrap enrichment analysis: [nullranges](https://bioconductor.org/packages/nullranges); <br> Gaussian processes for multi-omic time series; <br> Predictive modeling of gRNA efficiency | Celonis | <img width="100" src="../assets/wancenmu.jpg"> |
| [Sarah Reifeis](https://www.linkedin.com/in/sarah-reifeis-phd-5943b1157/) [@sarah_reifeis](https://twitter.com/sarah_reifeis) | Exposure effects with confounding | Eli Lilly | <img width="100" src="../assets/sarahreifeis.jpg"> |
| [Scott Van Buren](https://www.linkedin.com/in/scottvanburen/) | scRNA-seq quantification uncertainty | GSK | <img width="100" src="../assets/scottvanburen.jpg"> |
| [Achal Patel](https://www.linkedin.com/in/achal-patel-166733202?original_referer=https%3A%2F%2Fwww.google.com%2F) <br> [@pateap07](https://twitter.com/pateap07) | Breast cancer TWAS | Genentech | <img width="100" src="../assets/achalpatel.jpg"> |
| [Arjun Bhattacharya](https://bhattacharya-a-bt.github.io) [@bhattac_a_bt](https://twitter.com/bhattac_a_bt) | TWAS for breast cancer outcomes; Multi-omic strategies in TWAS: [MOSTWAS](https://bhattacharya-a-bt.github.io/MOSTWAS/); Compressed sensing for deconvolution: [DeCompress](https://bhattacharya-a-bt.github.io/DeCompress/) | [Assist. Prof., MD Anderson](https://bhattacharya-lab.com/) | <img width="100" src="../assets/arjunbhattacharya.jpg"> | 
| [Sean McCabe](https://mccabes292.github.io/) [@SeanMcCabeBIOS](https://twitter.com/SeanMcCabeBIOS) | Multi-omics visualization: [movie](https://github.com/mccabes292/movie); LDA of isoforms: [actor](https://github.com/mccabes292/actor) | [Assist. Prof., Indiana Univ.](https://medicine.iu.edu/faculty/62984/mccabe-sean) | <img width="100" src="../assets/seanmccabe.png"> |
| [Anqi Zhu](https://azhu513.github.io) <br> [@anqiz91](https://twitter.com/anqiz91) | Differential analysis in RNA-seq; Developed [apeglm](http://bioconductor.org/packages/apeglm), [swish](http://bioconductor.org/packages/fishpond), [mrlocus](https://mikelove.github.io/mrlocus) | Genentech | <img width="100" src="../assets/anqizhu.png"> |

### Masters and bachelors students

| Name | Research topic | Current position | <font color="#EEEEEE">~~~~~~~~~~</font> |
| :--- | :---- | :---- | :---- |
| [Isabella Gamez](https://www.linkedin.com/in/isabella-gamez-3766a01b5) | Bacterial RNA-seq - Rita Tamayo lab | NIEHS post-bac | <img width="100" src="../assets/isabellagamez.jpg"> |
| [Kwame Forbes](https://kwameforbes.github.io/) [@kwame_forbes](https://twitter.com/kwame_forbes) | Integration of single cell expression data for exploration of bulk DE analysis | UNC BCB | <img width="100" src="../assets/kwameforbes.jpg"> |
| Alexandre Lockhart | Identification of mutational signatures via targeted sequencing | Rho Inc. | <img width="100" src="../assets/alexlockhart.jpg"> |

## Collaborators

| Name | Research topic | <font color="#EEEEEE">~~~~~~~~~~</font> |
| :--- | :---- | :---- | :---- |
| Amanda Tapia | Colocalization of QTL and GWAS | <img width="100" src="../assets/amandatapia.jpg"> |
| Joshua Zitovsky | Fast implementation of *apeglm* for allelic expression ratios | <img width="100" src="../assets/joshuazitovsky.jpg"> | 


## Friends of the lab

(Current or former students from other labs that have contributed to Love Lab research)

| Name | Research topic | School (lab) | <font color="#EEEEEE">~~~~~~~~~~</font> |
| :--- | :---- | :---- | :---- |
| [Anna A. Monaco](https://de.linkedin.com/in/anna-alessandra-monaco-a01623141) <br> [@AnnaAMonaco](https://twitter.com/AnnaAMonaco) | Molecular Biology/Genetics/Evo-Devo | Robert Zinzen (Berlin) | <img width="100" src="../assets/annaamonaco.jpg"> |
| [Noor P. Singh](https://noorpratap.netlify.app/) <br> [@pratapnoor](https://twitter.com/pratapnoor) | Statistical methods for RNA-seq | UMD (Patro) | <img width="100" src="../assets/noorpratapsingh.jpg"> |
| [Dongze He](https://combine-lab.github.io/members/dongze-he.html) <br> [@dongzehe](https://twitter.com/dongzehe) | Single-cell RNA-seq | UMD (Patro) | <img width="100" src="../assets/dongzehe.jpg"> |
| [Avi Srivastava](https://k3yavi.github.io/) <br> [@k3yavi](https://twitter.com/k3yavi) | Fast mapping, scRNA-seq quant. | SBU (Patro) | <img width="100" src="../assets/avisrivastava.jpg"> |
| [Hirak Sarkar](https://hiraksarkar.github.io/) <br> [@hiraksarkar](https://twitter.com/hiraksarkar) | Selective alignment, <br> scRNA-seq simulation, <br> transcript grouping | UMD (Patro) | <img width="100" src="../assets/hiraksarkar.jpg"> |
| [Stuart Lee](https://stuartlee.org/) <br> [@_stuartlee](https://twitter.com/_stuartlee) | Human-centred data science, <br> tidy range operations | Melbourne | <img width="100" src="../assets/stuartlee.jpg"> |
