---
layout: page
title: "Papers"
---

### Publications and preprints

Gieira S. Jones, Katherine A. Hoadley, Halei Benefield, Linnea
T. Olsson, Alina M. Hamilton, **Arjun Bhattacharya**, Erin L. Kirk,
Heather J. Tipaldos, Jodie M. Fleming, Kevin P. Williams, 
**Michael I. Love**, Hazel B. Nichols, Andrew F. Olshan, 
Melissa A. Troester.
Racial differences in breast cancer outcomes by hepatocyte growth
factor pathway expression. 
*Breast Cancer Research and Treatment*, January 2022.
[doi: 10.1007/s10549-021-06497-w](https://doi.org/10.1007/s10549-021-06497-w)

Hannah J. Perrin, Kevin W. Currin, Swarooparani Vadlamudi, Gautam
K. Pandey, Kenneth K. Ng, Martin Wabitsch, Markku Laakso, 
**Michael I. Love**, Karen L. Mohlke.
Chromatin accessibility and gene expression during adipocyte
differentiation identify context-dependent effects at cardiometabolic
GWAS loci.
*PLOS Genetics*, October 2021.
[doi: 10.1371/journal.pgen.1009865](https://doi.org/10.1371/journal.pgen.1009865)

**Wancen Mu**, Hirak Sarkar, Avi Srivastava, Kwangbom Choi, Rob Patro,
**Michael I. Love**.
Airpart: Interpretable statistical models for analyzing allelic
imbalance in single-cell datasets.
*bioRxiv*, October 2021.
[doi: 10.1101/2021.10.15.464546](https://doi.org/10.1101/2021.10.15.464546),
[airpart](https://bioconductor.org/packages/airpart) R/Bioconductor package

Dan Liang, Nil Aygün, Nana Matoba, Folami Y. Ideraabdullah, 
**Michael I. Love**, Jason L. Stein.
Inference of cell-type specific imprinted regulatory elements and
genes during human neuronal differentiation.
*bioRxiv*, October 2021.
[doi: 10.1101/2021.10.04.463060](https://doi.org/10.1101/2021.10.04.463060)

Nicole E. Kramer, Eric S. Davis, Craig D. Wenger, Erika M. Deoudes, Sarah
M. Parker, **Michael I. Love**, Douglas H. Phanstiel.
Plotgardener: Cultivating precise multi-panel figures in R.
*bioRxiv*, September 2021.
[doi: 10.1101/2021.09.08.459338](https://doi.org/10.1101/2021.09.08.459338)

Hunyong Cho, Chuwen Liu, Boyang Tang, Bridget M. Lin, Jeffrey Roach,
Apoena de Aguiar Ribeiro, **Michael I. Love**, Kimon Divaris, Di Wu.
Distribution-based comprehensive evaluation of methods for
differential expression analysis in metatranscriptomics.
*bioRxiv*, July 2021.
[doi: 10.1101/2021.07.14.452374](https://doi.org/10.1101/2021.07.14.452374)

Nil Aygün, Angela L. Elwell, Dan Liang, Michael J. Lafferty, Kerry
E. Cheek, Kenan P. Courtney, Jessica Mory, Ellie Hadden-Ford, Oleh
Krupa, Luis de la Torre-Ubieta, Daniel H. Geschwind, **Michael I. Love**,
Jason L. Stein.
Brain-trait-associated variants impact cell-type-specific gene
regulation during neurogenesis.
*American Journal of Human Genetics*, September 2021.
[doi: 10.1016/j.ajhg.2021.07.011](https://doi.org/10.1016/j.ajhg.2021.07.011)

Gieira S. Jones, Katherine A. Hoadley, Linnea T. Olsson, Alina M.
Hamilton, **Arjun Bhattacharya**, Erin L. Kirk, Heather J. Tipaldos, Jodie M.
Fleming, **Michael I. Love**, Hazel B. Nichols, Andrew F. Olshan, Melissa A.
Troester
Hepatocyte growth factor pathway expression in breast cancer by race
and subtype.
*Breast Cancer Research*, August 2021.
[doi: 10.1186/s13058-021-01460-5](https://doi.org/10.1186/s13058-021-01460-5)

(Open Letter) Kim-Anh Lê Cao, Al J. Abadi, Emily F. Davis-Marcisak,
Lauren Hsu, Arshi Arora, Alexis Coullomb, Atul Deshpande, Yuzhou Feng,
Pratheepa Jeganathan, Melanie Loth, Chen Meng, **Wancen Mu**, Vera
Pancaldi, Kris Sankaran, Amrit Singh, Joshua S. Sodicoff, Genevieve
L. Stein-O’Brien, Ayshwarya Subramanian, Joshua D. Welch, Yue You,
Ricard Argelaguet, Vincent J. Carey, Ruben Dries, Casey S. Greene,
Susan Holmes, **Michael I. Love**, Matthew E. Ritchie, Guo-Cheng Yuan, 
Aedin C. Culhane, Elana Fertig.
Community-wide hackathons to identify central themes in single-cell multi-omics.
*Genome Biology*, August 2021.
[doi: 10.1186/s13059-021-02433-9](https://doi.org/10.1186/s13059-021-02433-9)

**Sean D. McCabe**, Andrew B. Nobel, **Michael I. Love**.
ACTOR: a latent Dirichlet model to compare expressed isoform
proportions to a reference panel. 
*Biostatistics*, May 2021.
[doi: 10.1093/biostatistics/kxab013](https://doi.org/10.1093/biostatistics/kxab013),
[actor](https://github.com/mccabes292/actor) R package

Dan Liang, Angela L. Elwell, Nil Aygün, Oleh Krupa, Justin M. Wolter,
Felix A. Kyere, Michael J. Lafferty, Kerry E. Cheek, Kenan
P. Courtney, Marianna Yusupova, Melanie E. Garrett, Allison
Ashley-Koch, Gregory E. Crawford, **Michael I. Love**, Luis de la
Torre-Ubieta, Daniel H. Geschwind, Jason L. Stein.
Cell-type specific effects of genetic variation on chromatin
accessibility during human neuronal differentiation.
*Nature Neuroscience*, May 2021.
[doi: 10.1038/s41593-021-00858-w](https://doi.org/10.1038/s41593-021-00858-w)

Ying Chen, Nadia M. Davidson, Yuk Kei Wan, Harshil Patel, Fei Yao,
Hwee Meng Low, Christopher Hendra, Laura Watten, Andre Sim, Chelsea
Sawyer, Viktoriia Iakovleva, Puay Leng Lee, Lixia Xin, Hui En Vanessa
Ng, Jia Min Loo, Xuewen Ong, Hui Qi Amanda Ng, Jiaxu Wang, Wei Qian
Casslynn Koh, Suk Yeah Polly Poon, Dominik Stanojevic, Hoang-Dai Tran,
Kok Hao Edwin Lim, Shen Yon Toh, Philip Andrew Ewels, Huck-Hui Ng,
N.Gopalakrishna Iyer, Alexandre Thiery, Wee Joo Chng, Leilei Chen,
Ramanuj DasGupta, Mile Sikic, Yun-Shen Chan, Boon Ooi Patrick Tan, Yue
Wan, Wai Leong Tam, Qiang Yu, Chiea Chuan Khor, Torsten Wüstefeld,
Ploy N. Pratanwanich, **Michael I. Love**, Wee Siong Sho Goh, Sarah B. Ng,
Alicia Oshlack, Jonathan Göke, SG-NEx consortium.
A systematic benchmark of Nanopore long read RNA sequencing for
transcript level analysis in human cell lines.
*bioRxiv*, April 2021.
[doi: 10.1101/2021.04.21.440736](https://doi.org/10.1101/2021.04.21.440736)

**Anqi Zhu** (1), Nana Matoba (1), Emmaleigh Wilson, Amanda L. Tapia, Yun Li,
Joseph G. Ibrahim, Jason L. Stein, **Michael I. Love**.
MRLocus: identifying causal genes mediating a trait through Bayesian
estimation of allelic heterogeneity.
*PLOS Genetics*, April 2021.
[doi: 10.1371/journal.pgen.1009455](https://doi.org/10.1371/journal.pgen.1009455),
[MRLocus](https://mikelove.github.io/mrlocus) R package

Anushka Rajesh, Yutong Chang, Malak S. Abedalthagafi, Annie
Wong-Beringer, **Michael I Love**, Serghei Mangul.
Improving the completeness of public metadata accompanying omics studies.
*Genome Biology*, April 2021.
[doi: 10.1186/s13059-021-02332-z](https://doi.org/10.1186/s13059-021-02332-z)
(Commentary)

Achal Patel, Montserrat García-Closas, Andrew F. Olshan, Charles
M. Perou, Melissa A. Troester, **Michael I. Love**, Arjun Bhattacharya.
Transcriptome-wide association study of risk of recurrence in Black
and White breast cancer patients.
*medRxiv*, March 2021.
[doi: 10.1101/2021.03.19.21253983](https://doi.org/10.1101/2021.03.19.21253983)

**Arjun Bhattacharya**, Yun Li, **Michael I. Love**.
Multi-omic strategies for transcriptome-wide prediction and association studies.
*PLOS Genetics*, March 2021.
[doi: 10.1371/journal.pgen.1009398](https://doi.org/10.1371/journal.pgen.1009398),
[MOSTWAS](https://bhattacharya-a-bt.github.io/MOSTWAS) R package

Hyo Young Choi, Heejoon Jo, Xiaobei Zhao, Katherine A. Hoadley, Scott
Newman, Jeremiah Holt, Michele C. Hayward, **Michael I. Love**,
J. S. Marron, D. Neil Hayes.
SCISSOR: a framework for identifying structural changes in RNA transcripts.
*Nature Communications*, January 2021.
[doi: 10.1038/s41467-020-20593-3](https://doi.org/10.1038/s41467-020-20593-3)

**Arjun Bhattacharya**, Alina M. Hamilton, Melissa A. Troester, **Michael I. Love**.
DeCompress: tissue compartment deconvolution of targeted mRNA
expression panels using compressed sensing.
*Nucleic Acids Research*, January 2021.
[doi: 10.1093/nar/gkab031](https://doi.org/10.1093/nar/gkab031),
[DeCompress](https://bhattacharya-a-bt.github.io/DeCompress) R package

**Scott Van Buren**, Hirak Sarkar, Avi Srivastava, Naim U. Rashid, Rob
Patro, **Michael I. Love**.
Compression of quantification uncertainty for scRNA-seq counts.
*Bioinformatics*, January 2021.
[doi: 10.1093/bioinformatics/btab001](https://doi.org/10.1093/bioinformatics/btab001),
`makeInfReps` in [fishpond](http://bioconductor.org/packages/fishpond) R/Bioconductor package

**Joshua P. Zitovsky**, **Michael I. Love**.
Fast effect size shrinkage software for beta-binomial models of
allelic imbalance. 
*F1000Research*, December 2020.
[doi: 10.12688/f1000research.20916.2](https://doi.org/10.12688/f1000research.20916.2),
`method="betabinCR"` in [apeglm](http://bioconductor.org/packages/apeglm) R/Bioconductor package

Nana Matoba, **Michael I. Love**, Jason L. Stein.
Evaluating brain structure traits as endophenotypes using polygenicity
and discoverability.
*Human Brain Mapping*, October, 2020.
[doi: 10.1002/hbm.25257](https://doi.org/10.1002/hbm.25257)

Halei C. Benefield, Katherine E. Reeder-Hayes, Hazel B. Nichols,
Benjamin C. Calhoun, **Michael I. Love**, Erin L. Kirk,
Joseph Geradts, Katherine A. Hoadley, Stephen R. Cole,
H. Shelton Earp, Andrew F. Olshan, Lisa A. Carey, Charles M.
Perou, Melissa A. Troester.
Outcomes of Hormone-Receptor Positive, HER2-Negative Breast Cancers by
Race and Tumor Biological Features.
*JNCI Cancer Spectrum*, September 2020.
[doi: 10.1093/jncics/pkaa072](https://doi.org/10.1093/jncics/pkaa072)

Miheer Dewaskar, John Palowitch, Mark He, **Michael I. Love**, Andrew Nobel.
Finding Stable Groups of Cross-Correlated Features in Multi-View data.
*arXiv*, September 2020.
[arXiv: 2009.05079](https://arxiv.org/abs/2009.05079)

Avi Srivastava, Laraib Malik, Hirak Sarkar, Mohsen Zakeri, Fatemeh Almodaresi,
Charlotte Soneson, **Michael I. Love**, Carl Kingsford, Rob Patro.
Alignment and mapping methodology influence transcript abundance estimation.
*Genome Biology*, September 2020.
[doi: 10.1186/s13059-020-02151-8](https://doi.org/10.1186/s13059-020-02151-8)

**Arjun Bhattacharya** (1), Alina M. Hamilton (1), Helena Furberg, Eugene Pietzak,
Mark P. Purdue, Melissa A. Troester, Katherine A. Hoadley (N), 
**Michael I. Love** (N).
An approach for normalization and quality control for NanoString RNA
expression data.
*Briefings in Bioinformatics*, August 2020.
[doi: 10.1093/bib/bbaa163](https://doi.org/10.1093/bib/bbaa163),
[PMC8138885](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8138885)

Hirak Sarkar, Avi Srivastava, Hector Corrada Bravo, 
**Michael I. Love**, Rob Patro.
Terminus enables the discovery of data-driven, robust transcript groups from RNA-seq data. 
*Bioinformatics*, July 2020.
[doi: 10.1093/bioinformatics/btaa448](https://doi.org/10.1093/bioinformatics/btaa448),
[Terminus](https://github.com/COMBINE-lab/terminus) software

**Sarah A. Reifeis**, Michael G. Hudgens, Mete Civelek, Karen
L. Mohlke, **Michael I. Love**. 
Assessing exposure effects on gene expression.
*Genetic Epidemiology*, June, 2020.
[doi: 10.1002/gepi.22324](https://doi.org/10.1002/gepi.22324),
[PMC7429346](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7429346)

Charlotte Soneson, Federico Marini, Florian Geier, **Michael
I. Love**, Michael B. Stadler.
ExploreModelMatrix: Interactive exploration for improved understanding
of design matrices and linear models in R.
*F1000Research*, June 2020.
[doi: 10.12688/f1000research.24187.1](https://doi.org/10.12688/f1000research.24187.1),
[ExploreModelMatrix](https://www.bioconductor.org/packages/ExploreModelMatrix) R/Bioconductor package

**Michael I. Love**, Charlotte Soneson, Peter F. Hickey, Lisa
K. Johnson, N. Tessa Pierce, Lori Shepherd, Martin Morgan, Rob Patro.
Tximeta: Reference sequence checksums for provenance identification in RNA-seq.
*PLOS Computational Biology*, February 2020.
[doi: 10.1371/journal.pcbi.1007664](https://doi.org/10.1371/journal.pcbi.1007664),
[tximeta](https://bioconductor.org/packages/tximeta) R/Bioconductor package

**Arjun Bhattacharya**, Montserrat Garcia-Closas, Andrew F. Olshan,
Charles M. Perou, Melissa A. Troester, **Michael I. Love**.
A framework for transcriptome-wide association studies in breast
cancer in diverse study populations.
*Genome Biology*, February 2020.
[doi: 10.1186/s13059-020-1942-6](https://doi.org/10.1186/s13059-020-1942-6)

Stuart Lee, Michael Lawrence, **Michael I. Love**.
Fluent genomics with plyranges and tximeta.
*F1000Research*, February 2020.
[doi: 10.12688/f1000research.22259.1](https://doi.org/10.12688/f1000research.22259.1),
[fluentGenomics workflow](https://bioconductor.org/packages/fluentGenomics)

Lourdes Cruz-Garcia, Grainne O'Brien, Botond Sipos, Simon Mayes,
**Michael I. Love**, Daniel J. Turner, Christophe Badie.
Generation of a Transcriptional Radiation Exposure Signature in Human
Blood Using Long-Read Nanopore Sequencing.
*Radiation Research*, December 2019.
[doi: 10.1667/RR15476.1](https://doi.org/10.1667/RR15476.1)

Chelsea K. Raulerson, Arthur Ko, John C. Kidd, Kevin W. Currin, Sarah
M. Brotman, Maren E. Cannon, Ying Wu, Cassandra N. Spracklen, Anne
U. Jackson, Heather M. Stringham, Ryan P. Welch, Christian
Fuchsberger, Adam E. Locke, Narisu Narisu, Aldons J. Lusis, Mete
Civelek, Terrence S. Furey, Johanna Kuusisto, Francis S. Collins,
Michael Boehnke, Laura J. Scott, Dan-Yu Lin, **Michael I. Love**,
Markku Laakso, Päivi Pajukanta, Karen L. Mohlke.
Adipose Tissue Gene Expression Associations Reveal Hundreds of
Candidate Genes for Cardiometabolic Traits.
*Americal Journal of Human Genetics*, October 2019.
[doi: 10.1016/j.ajhg.2019.09.001](https://doi.org/10.1016/j.ajhg.2019.09.001)

**Anqi Zhu**, Avi Srivastava, Joseph G. Ibrahim, Rob Patro, 
**Michael I. Love**.
Nonparametric expression analysis using inferential replicate counts.
*Nucleic Acids Research*, August 2019.
[doi: 10.1093/nar/gkz622](https://doi.org/10.1093/nar/gkz622),
[swish supplement PDF](../assets/swish_supplement.pdf),
`swish` function in the [fishpond](https://bioconductor.org/packages/fishpond) R/Bioconductor package

**Sean D. McCabe**, Dan-Yu Lin, **Michael I. Love**.
Consistency and overfitting of multi-omics methods on experimental data.
*Briefings in Bioinformatics*, July 2019.
[doi: 10.1093/bib/bbz070](https://doi.org/10.1093/bib/bbz070),
[PMC7373174](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7373174),
[movie](https://github.com/mccabes292/movie) R package

Charlotte Soneson,  **Michael I. Love**, Rob Patro, Shobbir Hussain,
Dheeraj Malhotra, Mark D. Robinson.
A junction coverage compatibility score to quantify the reliability of
transcript abundance estimates and annotation catalogs.
*Life Science Alliance*, January 2019.
[doi: 10.26508/lsa.201800175](https://doi.org/10.26508/lsa.201800175)

Lindsay A. Williams, Katherine A. Hoadley, Hazel B. Nichols, Joseph
Geradts, Charles M. Perou, **Michael I. Love**, Andrew F. Olshan, Melissa
A. Troester.
Differences in race, molecular and tumor characteristics among women
diagnosed with invasive ductal and lobular breast carcinomas.
*Cancer Causes & Control*, January 2019.
[doi: 10.1007/s10552-018-1121-1](https://doi.org/10.1007/s10552-018-1121-1)

Yoh Isogai, Zheng Wu, **Michael I.Love**, Michael Ho-Young Ahn,
Dhananjay Bambah-Mukku, Vivian Hua, Karolina Farrell, Catherine Dulac.
Multisensory Logic of Infant-Directed Aggression by Males.
*Cell*, December 2018.
[doi: 10.1016/j.cell.2018.11.032](https://doi.org/10.1016/j.cell.2018.11.032)

(Review) Koen Van Den Berge, Katharina Hembach, Charlotte Soneson, Simone
Tiberi, Lieven Clement, **Michael I. Love**, Rob Patro, Mark Robinson.
RNA sequencing data: hitchhiker's guide to expression analysis.
*Annual Review of Biomedical Data Science*, April 2019.
[doi: 10.1146/annurev-biodatasci-072018-021255](https://doi.org/10.1146/annurev-biodatasci-072018-021255),
<https://peerj.com/manuscripts/33040/>
(Review)

**Anqi Zhu**, Joseph G. Ibrahim, **Michael I. Love**.
Heavy-tailed prior distributions for sequence count data: removing the
noise and preserving large differences.
*Bioinformatics*, November 2018.
[doi: 10.1093/bioinformatics/bty895](https://doi.org/10.1093/bioinformatics/bty895),
[apeglm](http://bioconductor.org/packages/apeglm) R/Bioconductor package

**Michael I. Love**, Charlotte Soneson, and Rob Patro.
Swimming downstream: statistical analysis of differential transcript
usage following Salmon quantification.
*F1000Research*, October 2018.
[doi: 10.12688/f1000research.15398.3](https://doi.org/10.12688/f1000research.15398.3)
[rnaseqDTU workflow](https://bioconductor.org/packages/rnaseqDTU)

Yuchen Yang, Ruth Huh, Houston W. Culpepper, Yuan Lin, **Michael I. Love**, Yun Li.
SAFE-clustering: Single-cell Aggregated (From Ensemble) Clustering for Single-cell RNA-seq Data.
*Bioinformatics*, September 2018.
[doi: 10.1093/bioinformatics/bty793](https://doi.org/10.1093/bioinformatics/bty793)

Alena van Bömmel, **Michael I. Love**, Ho-Ryun Chung, Martin Vingron.
coTRaCTE predicts co-occurring transcription factors within cell-type specific enhancers.
*PLOS Computational Biology*, August 2018.
[doi: 10.1371/journal.pcbi.1006372](https://doi.org/10.1371/journal.pcbi.1006372)

Edward W. Pietryk, Kiristin Clement, Marwa Elnagheeb, Ryan Kuster,
Kayla Kilpatrick, **Michael I. Love**, Folami Y. Ideraabdullah.
Intergenerational response to the endocrine disruptor vinclozolin is
influenced by maternal genotype and crossing scheme.
*Reproducitive Toxicology*, March 2018.
[doi: 10.1016/j.reprotox.2018.03.005](https://doi.org/10.1016/j.reprotox.2018.03.005)

Koen Van den Berge (1), Fanny Perraudeau (1), Charlotte Soneson, 
**Michael I. Love**, Davide Risso, Jean-Philippe Vert, Mark
D. Robinson, Sandrine Dudoit (N), Lieven Clement (N).
Observation weights unlock bulk RNA-seq tools for zero inflation and
single-cell applications.
*Genome Biology*, February 2018.
[doi: 10.1186/s13059-018-1406-4](https://doi.org/10.1186/s13059-018-1406-4),
[ZINB-WaVE vignette](http://bioconductor.org/packages/zinbwave)

Lindsay A. Williams, Hazel B. Nichols, Katherine A. Hoadley, Chiu Kit
Tse, Joseph Geradts, Mary Elizabeth Bell, Charles M. Perou, 
**Michael I. Love**, Andrew F. Olshan, Melissa A. Troester.
Reproductive risk factor associations with lobular and ductal
carcinoma in the Carolina Breast Cancer Study. 
*Cancer Causes & Control*, November 2017.
[doi: 10.1007/s10552-017-0977-9](https://doi.org/10.1007/s10552-017-0977-9)

Doug H. Phanstiel, Kevin Van Bortle, Damek Spacek, Gaelen T. Hess,
Muhammad S. Shamim, Ido Machol, **Michael I. Love**, Erez L. Aiden, 
Michael C. Bassik, Michael P. Snyder.
Static and Dynamic DNA Loops form AP-1-Bound Activation Hubs during Macrophage Development.
*Molecular Cell*, September 2017.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610110/),
[doi: 10.1016/j.molcel.2017.08.006](https://doi.org/10.1016/j.molcel.2017.08.006)

Rob Patro, Geet Duggal, **Michael I. Love**, Rafael A. Irizarry, Carl Kingsford. 
Salmon provides fast and bias-aware quantification of transcript expression.
*Nature Methods*, March 2017.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600148/),
[doi: 10.1038/nmeth.4197](https://doi.org/10.1038/nmeth.4197),
[CMU PR](http://www.cbd.cmu.edu/gene_expression_analyses/),
[Salmon](https://combine-lab.github.io/salmon/) software

**Michael I. Love**, Matthew Huska, Marcel Jurk, Robert Schöpflin,
Stephan Starick, Kevin Schwahn, Samantha Cooper, Keith Yamamoto, 
Morgane Thomas-Chollier, Martin Vingron, Sebastiaan Meijsing.
Role of the chromatin landscape and sequence in determining cell
type-specific genomic glucocorticoid receptor binding and gene
regulation.
*Nucleic Acids Research*, November 2016.
[doi: 10.1093/nar/gkw1163](https://doi.org/10.1093/nar/gkw1163)

Leonardo Collado Torres, Abhinav Nellore, Alyssa C. Frazee, 
Christopher Wilks, **Michael I. Love**, Ben Langmead,
Rafael A. Irizarry, Jeffrey Leek, Andrew E. Jaffe.
Flexible expressed region analysis for RNA-seq with derfinder.
*Nucleic Acids Research*, October 2016.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5314792/),
[doi: 10.1093/nar/gkw852](https://doi.org/10.1093/nar/gkw852),
[derfinder](http://bioconductor.org/packages/derfinder) R package

---

Pre-<img src="../assets/unc.png"/> publications:

**Michael I. Love**, John B. Hogenesch, Rafael A. Irizarry. 
Modeling of RNA-seq fragment sequence bias reduces systematic errors
in transcript abundance estimation. 
*Nature Biotechnology*, September 2016.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5143225/),
[doi: 10.1038/nbt.3682](https://doi.org/10.1038/nbt.3682),
[blog post](https://mikelove.wordpress.com/2016/09/26/rna-seq-fragment-sequence-bias/),
[YouTube](https://www.youtube.com/watch?v=9xskajkNJwg),
[alpine](http://bioconductor.org/packages/alpine) R/Bioconductor package

Mingxiang Teng, **Michael I. Love**, Carrie A. Davis, Sarah Djebali,
Alexander Dobin, Brenton R. Graveley, Sheng Li, Christopher E. Mason,
Sara Olson, Dmitri Pervouchine, Cricket A. Sloan, Xintao Wei, Lijun
Zhan, Rafael A. Irizarry.
A benchmark for RNA-seq quantification pipelines. 
*Genome Biology*, April 2016.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4842274/),
[doi: 10.1186/s13059-016-0940-1](https://doi.org/10.1186/s13059-016-0940-1)

Charlotte Soneson, **Michael I. Love**, Mark D. Robinson. 
Differential analyses for RNA-seq: transcript-level estimates improve
gene-level inferences. 
*F1000Research*, December 2015. 
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4712774/),
[doi: 10.12688/f1000research.7563.1](https://doi.org/10.12688/f1000research.7563.1),
[tximport](http://bioconductor.org/packages/tximport) R/Bioconductor package

**Michael I. Love**, Simon Anders, Vladislav Kim, Wolfgang Huber. 
RNA-seq workflow: gene-level exploratory analysis and differential
expression. 
*F1000Research*, October 2015.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4670015/),
[doi: 10.12688/f1000research.7035.1](https://doi.org/10.12688/f1000research.7035.2),
[rnaseqGene workflow](http://www.bioconductor.org/help/workflows/rnaseqGene/)

Raman Kumar, *et al*.
Increased STAG2 dosage defines a novel cohesinopathy with intellectual
disability and behavioural problems. 
*Human Molecular Genetics*, October 2015.
[doi: 10.1093/hmg/ddv414](https://doi.org/10.1093/hmg/ddv414)

Stephan R. Starick, Jonas Ibn-Salem, Marcel Jurk, Céline Hernandez,
**Michael I. Love**, Ho-Ryun Chung, Martin Vingron, Morgane
Thomas-Chollier, Sebastiaan H. Meijsing. 
ChIP-exo signal associated with DNA-binding motifs provides insight
into the genomic binding of the glucocorticoid receptor and
cooperating transcription factors.
*Genome Research*, June 2015.
[doi: 10.1101/gr.185157.114](https://doi.org/10.1101/gr.185157.114)

Wolfgang Huber, Vincent J. Carey, Robert Gentleman, Simon Anders, Marc
Carlson, Benilton S. Carvalho, Hector Corrada C. Bravo, Sean Davis,
Laurent Gatto, Thomas Girke, Raphael Gottardo, Florian Hahne, Kasper
D. Hansen, Rafael A. Irizarry, Michael Lawrence, **Michael I. Love**,
James MacDonald, Valerie Obenchain, Andrzej K.  Oleś, Hervé Pagès,
Alejandro Reyes, Paul Shannon, Gordon K. Smyth, Dan Tenenbaum, Levi
Waldron, Martin Morgan. 
Orchestrating high-throughput genomic analysis with Bioconductor. 
*Nature Methods*, February 2015.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4509590/),
[doi: 10.1038/nmeth.3252](https://doi.org/10.1038/nmeth.3252)

Hao Hu, *et al*.
X-exome sequencing of 405 unresolved families
identifies seven novel intellectual disability genes. 
*Molecular Psychiatry*, February 2015.
[doi: 10.1038/mp.2014.193](https://doi.org/10.1038/mp.2014.193)

**Michael I. Love**, Wolfgang Huber, Simon Anders. 
Moderated estimation of fold change and dispersion for RNA-seq data
with DESeq2. 
*Genome Biology*, December 2014.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049/),
[doi: 10.1186/s13059-014-0550-8](https://doi.org/doi:10.1186/s13059-014-0550-8),
[DESeq2](http://bioconductor.org/packages/DESeq2) R/Bioconductor package

Wei Li, Han Xu, Tengfei Xiao, Le Cong, **Michael I. Love**, Feng
Zhang, Rafael A. Irizarry, Jun S. Liu, Myles Brown, X. S. Liu. 
MAGeCK enables robust identification of essential genes from
genome-scale CRISPR/cas9 knockout screens. 
*Genome Biology*, December 2014.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290824/),
[doi: 10.1186/s13059-014-0554-4](https://doi.org/10.1186/s13059-014-0554-4)

Jonas Ibn-Salem (1), Sebastian Köhler (1), **Michael I. Love**,
Ho-Ryun Chung, Ni Huang, Matthew E. Hurles, Melissa Haendel, Nicole
L. Washington, Damian Smedley, Christopher J. Mungall, Suzanna
E. Lewis, Claus-Eric Ott, Sebastian Bauer, Paul N. Schofield, Stefan
Mundlos, Malte Spielmann (N), Peter N. Robinson (N).
Deletions of chromosomal regulatory boundaries are associated with
congenital disease. 
*Genome Biology*, September 2014.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4180961/),
[doi: 10.1186/s13059-014-0423-1](https://doi.org/10.1186/s13059-014-0423-1)

Owen D. Solberg (1), Edwin J. Ostrin (1), **Michael I. Love**, Jeffrey
C. Peng, Nirav R. Bhakta, Lydia Hou, Christine Nguyen, Margaret Solon,
Cindy Nguyen, Andrea J. Barczak, Lorna T. Zlock, Denitza P. Blagev,
Walter E. Finkbeiner, K. Mark Ansel, Joseph R. Arron, David J. Erle (N),
Prescott G. Woodruff (N). 
Airway epithelial miRNA expression is altered in asthma. 
*American Journal of Respiratory and Critical Care Medicine*, November 2012.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530212/),
[doi: 10.1164/rccm.201201-0027OC](https://doi.org/10.1164/rccm.201201-0027OC)

Ruping Sun, **Michael I. Love**, Tomasz Zemojtel, Anne-Katrin Emde,
Ho-Ryun Chung, Martin Vingron, Stefan A. Haas. 
Breakpointer: using local mapping artifacts to support sequence
breakpoint discovery from single-end reads. 
*Bioinformatics*, April 2012.
[doi: 10.1093/bioinformatics/bts064](https://doi.org/10.1093/bioinformatics/bts064)

**Michael I. Love**, Alena Myšičková, Ruping Sun, Vera Kalscheuer,
Martin Vingron, Stefan A. Haas. 
Modeling read counts for CNV detection in exome sequencing data. 
*Statistical Applications in Genetics and Molecular Biology*, January 2011.
[PMC](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517018/),
[doi: 10.2202/1544-6115.1732](https://doi.org/10.2202/1544-6115.1732)

