---
layout: page
title: "Papers"
---

### Publications and preprints

<div class="alert">
<strong>Note:</strong> For a full list of publications, see 
<a href="https://scholar.google.com/citations?hl=en&user=vzXv764AAAAJ&view_op=list_works&sortby=pubdate">
this Google Scholar page</a>, or the PI's CV under <b>Lab members</b>.
The following partial list includes papers and preprints either led by members
of the lab, co-supervised work, or work with relevant software
contributions from members of the lab.
</div>

**Jonathan D Rosen**, K Alaine Broadaway, Sarah M Brotman, Karen L Mohlke, **Michael I Love**.
Higher eQTL power reveals signals that boost GWAS colocalization.
*bioRxiv* August 2025.
[doi: 10.1101/2025.08.05.668745](https://doi.org/10.1101/2025.08.05.668745)

**Yuriko Harigaya**, Nana Matoba, Brandon D. Le, Jordan M. Valone, Jason L. Stein, **Michael I. Love**, William Valdar.
Probabilistic classification of gene-by-treatment interactions on molecular count phenotypes.
Accepted at *PLOS Genetics*. January 2025. Preprint:
[doi: 10.1101/2024.08.03.605142](https://doi.org/10.1101/2024.08.03.605142)

**Sarah A. Reifeis**, Michael G. Hudgens, Melissa A. Troester, **Michael I. Love**.
Assessing Etiologic Heterogeneity for Multinomial Outcome with Two-Phase Outcome-Dependent Sampling Design. 
*American Journal of Epidemiology* July 2024.
[10.1093/aje/kwae212](https://doi.org/10.1093/aje/kwae212),
*medRxiv*: [doi: 10.1101/2022.07.20.22277805](https://doi.org/10.1101/2022.07.20.22277805)

**Wancen Mu**\*, Jiawen Chen\*, Eric S. Davis, Kathleen Reed, Douglas Phanstiel, **Michael I. Love**, Didong Li.
Gaussian Processes for Time Series with Lead-Lag Effects with Applications to Biology Data
Accepted at *Biometrics*. December 2024. Preprint:
[doi: 10.48550/arXiv.2401.07400](https://doi.org/10.48550/arXiv.2401.07400)

William J Hutchison, Timothy J Keyes, Helena L Crowell, Charlotte Soneson, **Wancen Mu**, **Ji-Eun Park**, Eric S Davis, Abdullah A Nahid, Ming Tang, Victor Yuan, Pierre-Paul Axisa, Jonathan W Kitt, Chi-Lam Poon, Noriaki Sato, Raphael Gottardo, Martin Morgan, Stuart Lee, Michael Lawrence, Stephanie C Hicks, Garry P Nolan, Kara L Davis, Anthony T Papenfuss, **Michael I Love**, Stefano Mangiola.
The tidyomics ecosystem: Enhancing omic data analyses.
*bioRxiv*, September 2023.
[doi: 10.1101/2023.09.10.557072](https://doi.org/10.1101/2023.09.10.557072)

* [Tidyomics organization page](https://github.com/tidyomics)
* Related work from the lab including:
  - [bootRanges](https://nullranges.github.io/nullranges/articles/bootRanges.html)
  - [matchRanges](https://nullranges.github.io/nullranges/articles/matchRanges.html)
  - [fluentGenomics](https://f1000research.com/articles/9-109/v1)
  - [tidy ranges tutorial](https://tidyomics.github.io/tidy-ranges-tutorial/)
  - [tidyomics workshop](https://tidyomics.github.io/tidyomicsWorkshopBioc2023/)

**Euphy Wu**, Noor P. Singh, Kwangbom Choi, Mohsen Zakeri, Matthew
Vincent, Gary A. Churchill, Cheryl L. Ackert-Bicknell, Rob Patro,
**Michael I. Love**.
SEESAW: detecting isoform-level allelic imbalance accounting for inferential uncertainty.
*Genome Biology*, July 2023.
[doi: 10.1186/s13059-023-03003-x](https://doi.org/10.1186/s13059-023-03003-x)

* `SEESAW` in [fishpond](https://mikelove.github.io/fishpond)
R/Bioconductor package

Noor Pratap Singh, **Michael I. Love**, and Rob Patro.
TreeTerminus —creating transcript trees using inferential replicate counts.
*iScience*, June 2023.
[doi: 10.1016/j.isci.2023.106961](https://doi.org/10.1016/j.isci.2023.106961)

**Wancen Mu**, Eric Davis, Stuart Lee, Mikhail Dozmorov, Douglas
H. Phanstiel, **Michael I. Love**.
bootRanges: Flexible generation of null sets of genomic ranges for
hypothesis testing.
*Bioinformatics*, April 2023.
[doi: 10.1093/bioinformatics/btad190](https://doi.org/10.1093/bioinformatics/btad190) <br/>

* `bootRanges` in [nullranges](https://nullranges.github.io/nullranges)
R/Bioconductor package

Eric S. Davis, **Wancen Mu**, Stuart Lee, Mikhail G. Dozmorov, 
**Michael I. Love**, Douglas H. Phanstiel.
matchRanges: Generating null hypothesis genomic ranges via
covariate-matched sampling.
*Bioinformatics*, April 2023.
[doi: 10.1093/bioinformatics/btad197](https://doi.org/10.1093/bioinformatics/btad197)

* `matchRanges` in [nullranges](https://nullranges.github.io/nullranges)
R/Bioconductor package


**Ji-Eun Park**, Markia A. Smith, Sarah C. Van Alsten, Andrea Walens,
Di Wu, Katherine A. Hoadley, Melissa A. Troester, **Michael I. Love**.
Diffsig: Associating Risk Factors With Mutational Signatures.
*bioRxiv*, February 2023.
[doi: 10.1101/2023.02.09.527740](https://doi.org/10.1101/2023.02.09.527740)

* [diffsig](https://github.com/jennprk/diffsig) R package

**Wancen Mu**, Hirak Sarkar, Avi Srivastava, Kwangbom Choi, Rob Patro,
**Michael I. Love**.
Airpart: Interpretable statistical models for analyzing allelic
imbalance in single-cell datasets.
*Bioinformatics*, April 2022.
[doi: 10.1093/bioinformatics/btac212](https://doi.org/10.1093/bioinformatics/btac212),

* [airpart](https://bioconductor.org/packages/airpart) R/Bioconductor package

**Achal Patel**, Montserrat García-Closas, Andrew F. Olshan, Charles
M. Perou, Melissa A. Troester, **Michael I. Love**, Arjun Bhattacharya.
Gene-Level Germline Contributions to Clinical Risk of Recurrence
Scores in Black and White Patients with Breast Cancer.
*Cancer Research*, January 2022.
[doi: 10.1158/0008-5472.CAN-21-1207](https://doi.org/10.1158/0008-5472.CAN-21-1207)

**Sean D. McCabe**, Andrew B. Nobel, **Michael I. Love**.
ACTOR: a latent Dirichlet model to compare expressed isoform
proportions to a reference panel. 
*Biostatistics*, May 2021.
[doi: 10.1093/biostatistics/kxab013](https://doi.org/10.1093/biostatistics/kxab013),

* [actor](https://github.com/mccabes292/actor) R package

**Anqi Zhu** (1), Nana Matoba (1), Emmaleigh Wilson, Amanda L. Tapia, Yun Li,
Joseph G. Ibrahim, Jason L. Stein, **Michael I. Love**.
MRLocus: identifying causal genes mediating a trait through Bayesian
estimation of allelic heterogeneity.
*PLOS Genetics*, April 2021.
[doi: 10.1371/journal.pgen.1009455](https://doi.org/10.1371/journal.pgen.1009455),

* [MRLocus](https://mikelove.github.io/mrlocus) R package

**Arjun Bhattacharya**, Yun Li, **Michael I. Love**.
Multi-omic strategies for transcriptome-wide prediction and association studies.
*PLOS Genetics*, March 2021.
[doi: 10.1371/journal.pgen.1009398](https://doi.org/10.1371/journal.pgen.1009398),

* [MOSTWAS](https://bhattacharya-a-bt.github.io/MOSTWAS) R package

**Arjun Bhattacharya**, Alina M. Hamilton, Melissa A. Troester, **Michael I. Love**.
DeCompress: tissue compartment deconvolution of targeted mRNA
expression panels using compressed sensing.
*Nucleic Acids Research*, January 2021.
[doi: 10.1093/nar/gkab031](https://doi.org/10.1093/nar/gkab031),

* [DeCompress](https://bhattacharya-a-bt.github.io/DeCompress) R package

**Scott Van Buren**, Hirak Sarkar, Avi Srivastava, Naim U. Rashid, Rob
Patro, **Michael I. Love**.
Compression of quantification uncertainty for scRNA-seq counts.
*Bioinformatics*, January 2021.
[doi: 10.1093/bioinformatics/btab001](https://doi.org/10.1093/bioinformatics/btab001),

* `makeInfReps` in [fishpond](https://mikelove.github.io/fishpond) R/Bioconductor package

**Joshua P. Zitovsky**, **Michael I. Love**.
Fast effect size shrinkage software for beta-binomial models of
allelic imbalance. 
*F1000Research*, December 2020.
[doi: 10.12688/f1000research.20916.2](https://doi.org/10.12688/f1000research.20916.2),

* `method="betabinCR"` in [apeglm](http://bioconductor.org/packages/apeglm) R/Bioconductor package

Nana Matoba, **Michael I. Love**, Jason L. Stein.
Evaluating brain structure traits as endophenotypes using polygenicity
and discoverability.
*Human Brain Mapping*, October, 2020.
[doi: 10.1002/hbm.25257](https://doi.org/10.1002/hbm.25257)

**Arjun Bhattacharya** (1), Alina M. Hamilton (1), Helena Furberg, Eugene Pietzak,
Mark P. Purdue, Melissa A. Troester, Katherine A. Hoadley (N), 
**Michael I. Love** (N).
An approach for normalization and quality control for NanoString RNA
expression data.
*Briefings in Bioinformatics*, August 2020.
[doi: 10.1093/bib/bbaa163](https://doi.org/10.1093/bib/bbaa163),
[PMC8138885](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8138885)

Hirak Sarkar, Avi Srivastava, Hector Corrada Bravo, 
**Michael I. Love**, Rob Patro.
Terminus enables the discovery of data-driven, robust transcript groups from RNA-seq data. 
*Bioinformatics*, July 2020.
[doi: 10.1093/bioinformatics/btaa448](https://doi.org/10.1093/bioinformatics/btaa448),

* [Terminus](https://github.com/COMBINE-lab/terminus) software

**Sarah A. Reifeis**, Michael G. Hudgens, Mete Civelek, Karen
L. Mohlke, **Michael I. Love**. 
Assessing exposure effects on gene expression.
*Genetic Epidemiology*, June, 2020.
[doi: 10.1002/gepi.22324](https://doi.org/10.1002/gepi.22324),
[PMC7429346](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7429346)

Charlotte Soneson, Federico Marini, Florian Geier, **Michael
I. Love**, Michael B. Stadler.
ExploreModelMatrix: Interactive exploration for improved understanding
of design matrices and linear models in R.
*F1000Research*, June 2020.
[doi: 10.12688/f1000research.24187.1](https://doi.org/10.12688/f1000research.24187.1),

* [ExploreModelMatrix](https://www.bioconductor.org/packages/ExploreModelMatrix) R/Bioconductor package

**Michael I. Love**, Charlotte Soneson, Peter F. Hickey, Lisa
K. Johnson, N. Tessa Pierce, Lori Shepherd, Martin Morgan, Rob Patro.
Tximeta: Reference sequence checksums for provenance identification in RNA-seq.
*PLOS Computational Biology*, February 2020.
[doi: 10.1371/journal.pcbi.1007664](https://doi.org/10.1371/journal.pcbi.1007664),

* [tximeta](https://bioconductor.org/packages/tximeta) R/Bioconductor package

**Arjun Bhattacharya**, Montserrat Garcia-Closas, Andrew F. Olshan,
Charles M. Perou, Melissa A. Troester, **Michael I. Love**.
A framework for transcriptome-wide association studies in breast
cancer in diverse study populations.
*Genome Biology*, February 2020.
[doi: 10.1186/s13059-020-1942-6](https://doi.org/10.1186/s13059-020-1942-6)

Stuart Lee, Michael Lawrence, **Michael I. Love**.
Fluent genomics with plyranges and tximeta.
*F1000Research*, February 2020.
[doi: 10.12688/f1000research.22259.1](https://doi.org/10.12688/f1000research.22259.1),

* [fluentGenomics workflow](https://bioconductor.org/packages/fluentGenomics)

**Anqi Zhu**, Avi Srivastava, Joseph G. Ibrahim, Rob Patro, 
**Michael I. Love**.
Nonparametric expression analysis using inferential replicate counts.
*Nucleic Acids Research*, August 2019.
[doi: 10.1093/nar/gkz622](https://doi.org/10.1093/nar/gkz622),
[swish supplement PDF](../assets/swish_supplement.pdf),

* `swish` function in [fishpond](https://mikelove.github.io/fishpond) R/Bioconductor package

**Sean D. McCabe**, Dan-Yu Lin, **Michael I. Love**.
Consistency and overfitting of multi-omics methods on experimental data.
*Briefings in Bioinformatics*, July 2019.
[doi: 10.1093/bib/bbz070](https://doi.org/10.1093/bib/bbz070),
[PMC7373174](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7373174),

* [movie](https://github.com/mccabes292/movie) R package

(Review) Koen Van Den Berge, Katharina Hembach, Charlotte Soneson, Simone
Tiberi, Lieven Clement, **Michael I. Love**, Rob Patro, Mark Robinson.
RNA sequencing data: hitchhiker's guide to expression analysis.
*Annual Review of Biomedical Data Science*, April 2019.
[doi: 10.1146/annurev-biodatasci-072018-021255](https://doi.org/10.1146/annurev-biodatasci-072018-021255),
<https://peerj.com/manuscripts/33040/>
(Review)

**Anqi Zhu**, Joseph G. Ibrahim, **Michael I. Love**.
Heavy-tailed prior distributions for sequence count data: removing the
noise and preserving large differences.
*Bioinformatics*, November 2018.
[doi: 10.1093/bioinformatics/bty895](https://doi.org/10.1093/bioinformatics/bty895),

* [apeglm](http://bioconductor.org/packages/apeglm) R/Bioconductor package

**Michael I. Love**, Charlotte Soneson, and Rob Patro.
Swimming downstream: statistical analysis of differential transcript
usage following Salmon quantification.
*F1000Research*, October 2018.
[doi: 10.12688/f1000research.15398.3](https://doi.org/10.12688/f1000research.15398.3)

* [rnaseqDTU workflow](https://bioconductor.org/packages/rnaseqDTU)

Koen Van den Berge (1), Fanny Perraudeau (1), Charlotte Soneson, 
**Michael I. Love**, Davide Risso, Jean-Philippe Vert, Mark
D. Robinson, Sandrine Dudoit (N), Lieven Clement (N).
Observation weights unlock bulk RNA-seq tools for zero inflation and
single-cell applications.
*Genome Biology*, February 2018.
[doi: 10.1186/s13059-018-1406-4](https://doi.org/10.1186/s13059-018-1406-4),

* [ZINB-WaVE vignette](http://bioconductor.org/packages/zinbwave)

Rob Patro, Geet Duggal, **Michael I. Love**, Rafael A. Irizarry, Carl Kingsford. 
Salmon provides fast and bias-aware quantification of transcript expression.
*Nature Methods*, March 2017.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600148/),
[doi: 10.1038/nmeth.4197](https://doi.org/10.1038/nmeth.4197),
[CMU PR](http://www.cbd.cmu.edu/gene_expression_analyses/),

* [Salmon](https://combine-lab.github.io/salmon/) software

**Michael I. Love**, Matthew Huska, Marcel Jurk, Robert Schöpflin,
Stephan Starick, Kevin Schwahn, Samantha Cooper, Keith Yamamoto, 
Morgane Thomas-Chollier, Martin Vingron, Sebastiaan Meijsing.
Role of the chromatin landscape and sequence in determining cell
type-specific genomic glucocorticoid receptor binding and gene
regulation.
*Nucleic Acids Research*, November 2016.
[doi: 10.1093/nar/gkw1163](https://doi.org/10.1093/nar/gkw1163)

---

Pre-<img src="../assets/unc.png"/> publications:

**Michael I. Love**, John B. Hogenesch, Rafael A. Irizarry. 
Modeling of RNA-seq fragment sequence bias reduces systematic errors
in transcript abundance estimation. 
*Nature Biotechnology*, September 2016.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5143225/),
[doi: 10.1038/nbt.3682](https://doi.org/10.1038/nbt.3682),
[blog post](https://mikelove.wordpress.com/2016/09/26/rna-seq-fragment-sequence-bias/),
[YouTube](https://www.youtube.com/watch?v=9xskajkNJwg),

* [alpine](http://bioconductor.org/packages/alpine) R/Bioconductor package

Charlotte Soneson, **Michael I. Love**, Mark D. Robinson. 
Differential analyses for RNA-seq: transcript-level estimates improve
gene-level inferences. 
*F1000Research*, December 2015. 
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4712774/),
[doi: 10.12688/f1000research.7563.1](https://doi.org/10.12688/f1000research.7563.1),

* [tximport](https://bioconductor.org/packages/tximport) R/Bioconductor package

**Michael I. Love**, Simon Anders, Vladislav Kim, Wolfgang Huber. 
RNA-seq workflow: gene-level exploratory analysis and differential
expression. 
*F1000Research*, October 2015.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4670015/),
[doi: 10.12688/f1000research.7035.1](https://doi.org/10.12688/f1000research.7035.2),

* [rnaseqGene workflow](https://www.bioconductor.org/packages/rnaseqGene)

Wolfgang Huber, Vincent J. Carey, Robert Gentleman, Simon Anders, Marc
Carlson, Benilton S. Carvalho, Hector Corrada C. Bravo, Sean Davis,
Laurent Gatto, Thomas Girke, Raphael Gottardo, Florian Hahne, Kasper
D. Hansen, Rafael A. Irizarry, Michael Lawrence, **Michael I. Love**,
James MacDonald, Valerie Obenchain, Andrzej K.  Oleś, Hervé Pagès,
Alejandro Reyes, Paul Shannon, Gordon K. Smyth, Dan Tenenbaum, Levi
Waldron, Martin Morgan. 
Orchestrating high-throughput genomic analysis with Bioconductor. 
*Nature Methods*, February 2015.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4509590/),
[doi: 10.1038/nmeth.3252](https://doi.org/10.1038/nmeth.3252)

**Michael I. Love**, Wolfgang Huber, Simon Anders. 
Moderated estimation of fold change and dispersion for RNA-seq data
with DESeq2. 
*Genome Biology*, December 2014.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049/),
[doi: 10.1186/s13059-014-0550-8](https://doi.org/doi:10.1186/s13059-014-0550-8),

* [DESeq2](https://bioconductor.org/packages/DESeq2) R/Bioconductor package

**Michael I. Love**, Alena Myšičková, Ruping Sun, Vera Kalscheuer,
Martin Vingron, Stefan A. Haas. 
Modeling read counts for CNV detection in exome sequencing data. 
*Statistical Applications in Genetics and Molecular Biology*, January 2011.
[PMC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517018/),
[doi: 10.2202/1544-6115.1732](https://doi.org/10.2202/1544-6115.1732),

* [exomeCopy](https://bioconductor.org/packages/exomeCopy) R/Bioconductor package
